control-plane/rest/openapi-specs/v0_api_spec.yaml

```
@@ -2978,6 +3028,37 @@ components:
       required:
         - spec
         - state
+    Snapshot:
+      description: |-
+        Snapshot Information
+      type: object
+      properties:
+        uuid:
+          $ref: '#/components/schemas/SnapshotId'
+        name:
+          description: Name of the snapshot
+          type: string
+        size:
+          description: Size that the snapshot should be.
+          type: integer
+          format: int64
+          minimum: 0
+        source:
```

```suggestion
        source_volume:
```
Just to make it clear?